src/Stratis.Bitcoin/P2P/Peer/NetworkPeer.cs

```diff
@@ -989,159 +987,6 @@ public override string ToString()
             return string.Format("{0} ({1})", this.State, this.PeerAddress.Endpoint);
         }
 
-
-        /// <summary>
-        /// Get the chain of headers from the peer (thread safe).
-        /// </summary>
-        /// <param name="hashStop">The highest block wanted.</param>
-        /// <param name="cancellationToken"></param>
-        /// <returns>The chain of headers.</returns>
-        public ConcurrentChain GetChain(uint256 hashStop = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
All this has moved to the integration tests, is it not used in the code base but for testing?

Reply:
yes, this does not belong to NP class, this is just something a couple of tests need and that's it
after all, the solution compiles, tests pass ...